ci/prebuild.yml

```diff
@@ -15,30 +21,35 @@ style prebuild:
   script:
     - nix-shell --run "poetry run make style_check"
 
+# Check validity of coin definitions and protobuf files
 common prebuild:
   stage: prebuild
   <<: *gitlab_caching
   script:
     - nix-shell --run "poetry run make defs_check"
 
+# Check validity of auto-generated files
 gen prebuild:
   stage: prebuild
   <<: *gitlab_caching
   script:
     - nix-shell --run "poetry run make gen_check"
```

Comment:
I think `generate_docs.py` should be added to the `gen` and `gen_check targets to make regenerating and catching outdated docs easy. Probably conflicts with using the actual git revision, let's leave master there.